src/util/logger.js

```diff
@@ -0,0 +1,22 @@
+import pino from 'pino';
+
+
+const isDev = process.env.NODE_ENV !== 'production';
```

Comment:
Such a nitpick, but to me `test` environment is different than `development` environment, so can we have a `isProd` flag instead of an `isDev` here?